src/orcania.c

```diff
@@ -264,6 +264,19 @@ static char *strnstr(const char *haystack, const char *needle, size_t len) {
 }
 #endif
 
+#ifdef WIN
+static const char *strcasestr(const char *s1, const char *s2) {
```

Comment:
I prefer keeping the explicit parameter names, even if it's a local static function, better for future maintenance.

```C
static const char *strcasestr(const char *haystack, const char *needle) {
```

Reply:
I'm going to fix that ...